### ISSUE TITLE
Installation impossible : Error 404 on login

### ISSUE THREAD

**Issue by cercec** [directus/directus#2662]
# Bug Report
Since this morning I am trying to install Directus and it failed every time with the same error. I have tried the installation both locally with a basic LAMP server and with OneClick installation on Digital Ocean. 

## Steps to Reproduce

1. Create a droplet with Directus OneClick in the marketplace or clone the git repository
2. Connect with SSH and get the Directus mysql credentials
3. Visit the admin URL and start the configuration with the credentials
4. Once the form is complete, hit INSTALL button and wait for the green notification saying that the API was well configured.
5. Try to login : there is a message warning "Something is wrong with this project" with a 404 error.

## Expected Behavior
You should login with no problem with the admin credentials you just set up

## Actual Behavior

Get the message : "Something is wrong with this project"

In the console, the error 404 is for the API

 
## Other Context & Screenshots

<img width="1491" alt="Capture d’écran 2020-04-12 à 11 57 12" src="https://user-images.githubusercontent.com/60812978/79066011-ccca7800-7cb4-11ea-8e7a-b2d1900e3280.png">

## Technical Details

- Device: Desktop
- OS: MacOS 10.15.4 / Ubuntu 18.04
- Web Server: Apache/2.4.29
- PHP Version: 7.2.24
- Database: MySQL 5.7.29
- Directus Release Version: 8.6.1 on Digital Ocean / 8.6.2 on Local
  - App Version: ?
  - API Version: ?
- Install Method:
  - Oneclick Installation on Digital Ocean / Git clone on Local

**Comment by rijkvanzanten** [comment]
Do you have mod_rewrite enabled on your server? All requirements met?

**Comment by cercec** [comment]
Yes all requirements are met. (On Oneclick Installation with Digital Ocean, these requirements are full-filed by DO)

**Comment by rijkvanzanten** [comment]
Did you rename or change any of the source files or config after setting it up with the one-click? I notice your admin is located on `/admin-resistic`?

**Comment by cercec** [comment]
I didn't rename anything manually. This name comes from the Project Name and Project Key inputs in the configuration form.

**Comment by cercec** [comment]
Hello, I tried to go through a fresh installation on Ubuntu 18.04 with php 7.2 and Mariadb 5.7. Still no luck. Everything goes smoothly till the login page 

<img width="844" alt="Capture d’écran 2020-04-14 à 22 34 22" src="https://user-images.githubusercontent.com/60812978/79271986-0a8bf400-7ea1-11ea-81b0-ff449b9a42f9.png">

<img width="633" alt="Capture d’écran 2020-04-14 à 22 38 49" src="https://user-images.githubusercontent.com/60812978/79272007-14155c00-7ea1-11ea-9a41-328116d5b5ac.png">

**Comment by benhaynes** [comment]
Docs state MariaDB 10.2+ ... please ensure your server meets the minimum requirements.

https://docs.directus.io/installation/git.html#step-1-make-sure-requirements-are-met

**Comment by cercec** [comment]
Oh sorry for that, that was a typo. My server is running Mariadb 10.4.12

**Comment by cercec** [comment]
Tried to set up a new project with a new database. Same problem as before : the project seems correctly configured but get an error 404 when I want to login : 

<img width="713" alt="Capture d’écran 2020-04-14 à 23 59 12" src="https://user-images.githubusercontent.com/60812978/79278476-0c0ee980-7eac-11ea-85f3-7684e9a6acd4.png">

<img width="1767" alt="Capture d’écran 2020-04-15 à 00 01 01" src="https://user-images.githubusercontent.com/60812978/79278575-3e204b80-7eac-11ea-81e7-2abd96fa8260.png">
